lib/bas/utils/operaton/external_task_client.rb

```
@@ -3,113 +3,137 @@
+module Utils
+  module Operaton
+    # Client for interacting with Operaton's External Task API
+    #
+    # This client provides methods to manage external task lifecycle including:
+    # - Fetching and locking tasks
+    # - Completing tasks with variables
+    # - Unlocking tasks
+    # - Reporting task failures
+    #
+    # @example
+    #   client = Utils::Operaton::ExternalTaskClient.execute(base_url: "https://api.operaton.com", worker_id: "worker-123")
```

RuboCop (1.75.5)

[convention] Line is too long. [123/120]

(Layout/LineLength)